ui/guest/src/classes/ICERegistry.ts

```diff
@@ -218,12 +221,12 @@ export class ICERegistry {
       .map((rec) => rec.id);
   }
 
-  getComponentItemReceptacles(record) {
+  getComponentItemReceptacles(record: Record) {
     const contentType = this.getReferentialEntries(record).contentType;
     return this.getContentTypeReceptacles(contentType).map((rec) => rec.id);
   }
 
-  getContentTypeReceptacles(contentType) {
+  getContentTypeReceptacles(contentType: string | ContentType) {
```

Comment:
- Return type
- Declare overrides (do this for all that accept different input types)